assets/scripts/DocsNav.yml

```
@@ -32,18 +32,21 @@ jobs:
 
           # Define the URL of the navbar to be used
           NAVBAR_URL="https://raw.githubusercontent.com/TuringLang/turinglang.github.io/main/assets/scripts/TuringNavbar.html"
-         
-          # Update all HTML files in the current directory (gh-pages root)
-          ./insert_navbar.sh . $NAVBAR_URL
-         
+          
+          # Define file & folder to exclude (comma-separated list), Un-Comment the below line for excluding anything!
+          # EXCLUDE_PATHS="file.extension,folder,nested/folder,nestes/file.extension"
```

```suggestion
          # Define file & folder to exclude (comma-separated list), Un-Comment the below line for excluding anything!
          # export EXCLUDE_PATHS="file.extension,folder,nested/folder,nestes/file.extension"
```

Otherwise the next line won't pick it up.

---

It was working fine without export, Let me test it with this too!

---

Oh, hmm, you're right :) I didn't realise that all the lines get run as a single script, rather than as separate commands!

---

Yeah, so I guess we this PR is good to go!